test/common/common/log_macros_test.cc

```diff
@@ -330,6 +330,25 @@ TEST(FineGrainLog, SetLevel) {
   EXPECT_EQ(getFineGrainLogContext().getFineGrainLogEntry(__FILE__)->level(), spdlog::level::info);
 }
 
+TEST(FineGrainLog, SetJsonEscapedFormat) {
```

Comment:
we should test the contents logged for escaped or have https://github.com/envoyproxy/envoy/blob/6647c3f7d22bf8ab080f3b3d6ab4fb42cbb14629/test/common/common/logger_test.cc#L117 run parameterized with a logger created from the fine grain logger. I think the second approach is better.

Reply:
Done. Thank you!